components/session-service/Makefile

```diff
@@ -29,8 +29,7 @@ start: build
 	./bin/session-service examples/config.yml
 
 test:
-	@go test -v -i $(packages)
-	@go test -v $(packages) -cover
+	@go test -v -count=1 $(packages) -cover
```

Comment:
drive-by: `count=1` ensures you don't see cached tests results. Might just be me who doesn't appreciate that 😄